git_theta/merges/average.py

```diff
@@ -0,0 +1,80 @@
+"""Merge parameters by averaging them."""
+
+from typing import Dict, Any
+from git_theta import metadata
+from git_theta.merges import Merge
+from git_theta.utils import DiffState
+from git_theta.types import ParamName
+from git_theta import updates
+from git_theta import params
+from git_theta import async_utils
+from git_theta import git_utils
+
+
+PartialModel = Dict[ParamName, Any]
+Parameter = Any
+
+
+# TODO: Add a configurable interpolation parameter.
+class Average(Merge):
+    DESCRIPTION = "Average our change and their change together."
+    NAME = "average"
+    SHORT_CUT = "avg"
+    INACTIVE_STATES = {
+        # Averaging with the original parameter doesn't make sense?
+        DiffState.CHANGED_A,
+        DiffState.CHANGED_B,
+        # Averaging with only one added parameter doesn't make sense.
+        DiffState.ADDED_A,
+        DiffState.ADDED_B,
+        # Averaging with a deleted parameter doesn't make sense
+        DiffState.DELETED_A,
+        DiffState.DELETED_B,
+        DiffState.DELETED_BOTH,
+    }
+
+    def average(self, a: Parameter, b: Parameter) -> Parameter:
```

Comment:
Can this be parameterized so that we can support interpolation

Can push this to another issue for later

Reply:
This has been updated to work on variable number of params, but not interpolate yet. I did leave a TODO on adding interpolation tho.